INSTALL.adoc

```diff
@@ -46,8 +46,7 @@ Examples:
 * On a Linux x86-64 host, to build a 32-bit version of OCaml:
 
     ./configure --build=x86_64-pc-linux-gnu --host=i386-linux \
-                CC='gcc -m32' AS='as --32' ASPP='gcc -m32 -c' \
-                PARTIALLD='ld -r -melf_i386'
+                CC='gcc -m32' PARTIALLD='ld -r -melf_i386'
```

Comment:
This looks like a nice simplification.  However, I'm not sure it works.  Since the C compiler is gcc, and the target architecture i386, it looks like the configure script will take `default_as="${toolpref}as"`, missing the crucial `--32` flag that tells the assembler to produce 32-bit code.  I think you need to keep `AS='as --32'` here.  Getting rid of ASPP looks fine, though.